### PR TITLE
preload png icons for wizard

### PR DESCRIPTION
**Pull request by barshimi**
Close: #5718

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx [hunk 1]
```diff
@@ -155,6 +156,9 @@ const CREATE_SERVICE_STEPS: WizardStep[] = [
     stepName: "CreateServiceNextSteps",
   },
 ];
+
+const PLUGIN_LOGO_BASE_URL =
+  "https://raw.githubusercontent.com/amplication/plugin-catalog/master/assets/icons/";
```
Comment (abrl91) [inline packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx:161]: What is that?

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceDatabase.tsx [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
-  "https://raw.githubusercontent.com/amplication/plugin-catalog/master/assets/icons/";
+import "../CreateServiceWizard.scss";
 
-const PostgresPng = ImgSvg(`${PLUGIN_LOGO_BASE_URL}db-postgres.png`, "large");
-const MongoPng = ImgSvg(`${PLUGIN_LOGO_BASE_URL}db-mongo.png`, "large");
-const MysqlPng = ImgSvg(`${PLUGIN_LOGO_BASE_URL}db-mysql.png`, "large");
+interface Props extends WizardStepProps {
+  PostgresPng: React.ReactElement<any, any>;
+  MongoPng: React.ReactElement<any, any>;
+  MysqlPng: React.ReactElement<any, any>;
```
Comment (abrl91) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceDatabase.tsx:11]: can we use types here?

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/ImgSvg.tsx [hunk 3]
```diff
@@ -1,9 +1,34 @@
+import React, { useEffect, useState } from "react";
 import classNames from "classnames";
 
-const ImgSvg = (
+export const useImageLoader = (currentSrc: string) => {
+  const [imageSrc, _setImageSrc] = useState(null);
```
Comment (abrl91) [inline packages/amplication-client/src/Resource/create-resource/wizard-pages/ImgSvg.tsx:5]: What does `_setImageSrc` mean?

File: packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx (hunk 1 above)
Comment (barshimi) [reply packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.tsx:161]: this is the github icon url

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceDatabase.tsx (hunk 2 above)
Comment (barshimi) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceDatabase.tsx:11]: yes we can

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/ImgSvg.tsx (hunk 3 above)
Comment (barshimi) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/ImgSvg.tsx:5]: its a set function of useState. sometime it's recommended to add _ before set

**Review by morhag90 — APPROVED** [review]

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceDatabase.tsx (hunk 2 above)
Comment (abrl91) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceDatabase.tsx:11]: so why do we use any?

File: packages/amplication-client/src/Resource/create-resource/wizard-pages/ImgSvg.tsx (hunk 3 above)
Comment (abrl91) [reply packages/amplication-client/src/Resource/create-resource/wizard-pages/ImgSvg.tsx:5]: But we are not using it anywhere. It's not our convention 🙈

**Review by abrl91 — APPROVED** [review]

**Pull request merged**